app/src/main/res/layout-land/content_onboarding_default_browser.xml

```diff
@@ -75,27 +96,4 @@
         app:layout_constraintTop_toBottomOf="@id/browserProtectionTitle"
         app:layout_constraintWidth_max="500dp" />
 
-    <Button
-        android:id="@+id/continueButton"
-        android:layout_width="0dp"
-        android:layout_height="wrap_content"
-        android:text="@string/onboardingContinue"
-        style="@style/OnboardingButtonPrimaryCta"
-        app:layout_constraintBottom_toTopOf="@+id/launchSettingsButton"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent" />
-
-
-    <Button
-        android:id="@+id/launchSettingsButton"
-        android:layout_width="0dp"
-        android:layout_height="wrap_content"
-        style="@style/OnboardingButtonSecondaryCta"
-        android:text="@string/onboardingDefaultBrowserGoToSettingsAction"
```

Comment:
`onboardingDefaultBrowserGoToSettingsAction` and `defaultBrowserWelcome` are no longer used, can you delete them (and any translations). `

Reply:
Thanks! All done!